### ISSUE TITLE
Confusion regarding evaluation metrics in logs: val_AP_video vs val_AP_frame?

### ISSUE THREAD

**Issue by geekHedgy** [realtime-intelligence/evrt-detr#13]
Hi authors,

First of all, congratulations on the acceptance to `ICCV`! It is an inspiring work.

I have downloaded the pretrained `ResNet-18` model and training logs for the Gen4 (1Mpx) dataset to reproduce the results. I noticed a discrepancy that confuses me:

In the README.md, the evaluation command for the second stage uses `--data-name video`, which seemingly corresponds to the `AP_video` metric.

However, when I checked the provided `history_val.csv` (training log), I found that the `val_AP_video` is actually lower than the `val_AP ` achieved in `Stage 1 `.

On the contrary, the `val_AP_frame` in the log is higher and seems to match the expected performance improvements reported in the paper.

Could you please clarify which metric is the standard one reported in the paper (Table 1)? Should we primarily rely on `val_AP_frame` to evaluate the model's performance?

Thank you for your help!

Best regards.
